release-notes/8.0/known-issues.md

```diff
@@ -50,3 +50,11 @@ For details about known issues, please refer to the individual repositories:
 ## Native AOT Support in .NET
 
 For details about known issues, please reference to [the pinned issue](https://github.com/dotnet/core/issues/8288) in this repo.
+
+## WPF [ 8.0.100-preview.6.] `dotnet build` will fail for WPF projects having `IntermediateOutputPath` redirected or `IncludePackageReferencesDuringMarkupCompilation` set to false
+Available Workarounds:
```

Comment:
Let's pull the details of the error in here for better search engine optimization:

````suggestion
When building XAML files, you may encounter an error like

```
S:\FxWpf\obj\net472\MainWindow.g.cs(57,21): error CS1504: Source file 'MainWindow.xaml' could not be opened -- Could not find file. [S:\FxWpf\FxWpf\FxWpf_mrhd1mq3_wpftmp.csproj]
```

Where the error arises from a `*_wpftmp.csproj` project, is `CS1504`, and references a `.xaml` file.

Available Workarounds:
````